Inc/DDSTextureLoader.h

```diff
@@ -40,6 +40,14 @@ namespace DirectX
     };
 #endif
 
+    enum DDS_LOADER_FLAGS : uint32_t
+    {
+        DDS_LOADER_DEFAULT = 0,
+        DDS_LOADER_FORCE_SRGB = 0x1,
+        DDS_LOADER_IGNORE_SRGB = 0x2,
+        DDS_LOADER_SRGB_DEFAULT = 0x4,
```

Comment:
I don't think we need ``DDS_LOADER_SRGB_DEFAULT``, so remove it.

Reply:
I'll remove this extra flag and merge.

Reply:
Sounds good.  Sorry about the response delay, this is an off-hours task for me.